gravitee-am-ui/src/app/settings/domains/domains.component.html

```diff
@@ -45,13 +45,22 @@ <h1 *ngIf="!isEmpty">Domains</h1>
     </ngx-datatable>
   </div>
 
-  <app-emptystate *ngIf="isEmpty" [message]="'Domains will appear here'"
-                  [subMessage]="'Start creating a domain'"
-                  [icon]="'domain'"></app-emptystate>
-
-  <div *hasPermission="['domain_create']" [ngClass]="{'gv-add-button': !isEmpty, 'gv-add-button-center': isEmpty}">
-    <a mat-fab color="primary" [routerLink]="['new']">
-      <mat-icon>add</mat-icon>
-    </a>
+  <div *ngIf="!hasEnv">
```

Comment:
Can we display this div block ? Is it possible to access this page without a env ?

Reply:
you're right, it is not possible to list domains without an environment so no chance to access this page in that case.
I'll simplify the whole page.